bindings/python/src/torrent_info.cpp

```diff
@@ -307,6 +307,14 @@ void bind_torrent_info()
 
     class_<announce_entry>("announce_entry", init<std::string const&>())
         .def_readwrite("url", &announce_entry::url)
+        .def_readonly("trackerid", &announce_entry::trackerid)
+        .def_readonly("message", &announce_entry::message)
+        .def_readonly("last_error", &announce_entry::last_error)
+        .def_readonly("next_announce", &announce_entry::next_announce)
+        .def_readwrite("min_announce", &announce_entry::min_announce)
```

Comment:
they should all be read-only. iirc, this is the response from the tracker, saying you should not re-announce before this time